src/components/form/form.scss

```diff
@@ -10,12 +10,112 @@
     }
 }
 
+.limel-form-layout--default {
+    --gap: 1rem;
+    display: grid;
```

Comment:
Putting this here, so we can have a thread:

> (Kiarokh) Another thing I came to think of that we should test in the new form layouts is `limel-select`, which lets you choose multiple choices. The picture below is from another branch and this behaviour (expanding the column) may not happen in our layout branch. But we must test this.
> ![image](https://user-images.githubusercontent.com/35954987/100874292-a60bbc80-34a4-11eb-88bd-0bdc5bbd094d.png)

Confirmed. I just hard-coded the width of a textarea, and it squashed the other columns:

![image](https://user-images.githubusercontent.com/154725/101146209-04f34200-361b-11eb-8e77-cc71dc39f9f3.png)

Hopefully there's something similar to `flex-grow: 0` we can use to stop that from happening… I'm looking at it now.

Reply:
⚡ 

See line 29 in this file:

```scss
    // Using `minmax(0, 1fr)` below, instead of just `1fr` ensures that
    // components that are wider than its column doesn't make the column grow
    // and squash the other columns. /Ads
    grid-template-columns: repeat(var(--number-of-columns), minmax(0, 1fr));
```

Reply:
I haven't looked at what happens with a select with loads of options selected, but if there's a problem, it's something that will have to be fixed in limel-select (or whatever other component might be affected), and I don't think we have to fix any such issues in _this_ PR.

Reply:
you're right. Let's make new issues for them to fix.